public/src/channel/channel-page.html

```diff
@@ -514,25 +489,21 @@
 
       _refreshFeed() {
         this.items = [];
-        this.promotedItems = [];
         this._promotedCardIds = [];
         this._moreAvailable = false;
         this._lastCardId = null;
         this._fetchTimestamp = (new Date()).getTime();
         return $core.getFeed("channel", 60, null, null, this._handle, []).then((feed) => {
           let list = [];
-          let plist = [];
           for (let i = 0; i < feed.cards.length; i++) {
+            list.push(feed.cards[i]);
```

Comment:
Perhaps we should go back and revise the APIs to separate the promoted cards into a separate array.  There a bunch of extra work on the server side to position them carefully which is now unnecessary.  Fine for now, but a potential cleanup item later.

Reply:
Definitely. We are already iterating the cards to create the "_promotedCardIds" list for the more call